delivery_state/i18n/es.po

```diff
@@ -16,6 +16,84 @@ msgstr ""
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
+#. module: delivery_state
+#: model:mail.template,body_html:delivery_state.delivery_notification
+msgid ""
+"\n"
+"<div>\n"
+"    % set partner = object.sale_id.partner_id or object.partner_id\n"
+"    <p>\n"
+"        Hello, ${partner.name}.\n"
+"    </p>\n"
+"    <p>\n"
+"        We are glad to inform you that your order has been shipped by ${object.carrier_id.name}.\n"
+"        %if object.carrier_tracking_ref:\n"
+"            Your tracking reference is\n"
+"            <strong>\n"
+"            %if object.carrier_tracking_url:\n"
+"                % set multiple_carrier_tracking = object.get_multiple_carrier_tracking()\n"
+"                %if multiple_carrier_tracking:\n"
+"                    % for line in multiple_carrier_tracking:\n"
+"                        <br/><a href=\"${line[1]}\" target=\"_blank\">${line[0]}</a>\n"
+"                    % endfor\n"
+"                %else:\n"
+"                    <a href=\"${object.carrier_tracking_url}\" target=\"_blank\">${object.carrier_tracking_ref}</a>.\n"
+"                %endif\n"
+"            %else:\n"
+"                ${object.carrier_tracking_ref}.\n"
+"            %endif\n"
+"            </strong>\n"
+"        %endif\n"
+"        <br/><br/>\n"
+"        Thank you,<br/>\n"
+"        % if user and user.signature:\n"
+"          ${user.signature | safe}\n"
+"        % endif\n"
+"    </p>\n"
+"</div>\n"
+"\n"
+"        "
+msgstr ""
+"\n"
+"<div>\n"
+"    % set partner = object.sale_id.partner_id or object.partner_id\n"
+"    <p>\n"
+"        Hola, ${partner.name}.\n"
+"    </p>\n"
+"    <p>\n"
+"        Nos complace informarle de que su pedido ha sido enviado por ${object.carrier_id.name}.\n"
```

Comment:
```suggestion
"        Nos complace informarle que su pedido ha sido enviado por ${object.carrier_id.name}.\n"
```

Reply:
https://www.fundeu.es/recomendacion/queismo/

Reply:
Right, I always fall in the same one:

https://forum.wordreference.com/threads/informar-informarle-de-que.2470670/

That's why foreign students hate Spanish...